### ISSUE TITLE
[Desktop] Explain Quota in FAQ

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/docs-client-desktop#49]
## WHAT Needs to be Documented?

![Screenshot_20210518_093414-1](https://user-images.githubusercontent.com/214010/118618037-80a90b80-b7c3-11eb-8111-37927ec2d467.png)

first number:
- data used in my quota
- plus data used in external storages
- plus data used in other users shares

Second number:
- my quota
- plus data used in external storages
- plus data used in other users shares

## WHERE Does This Need To Be Documented (Link)?

https://doc.owncloud.com/desktop/faq.html

## WHY Should This Change Be Made?

We sometimes get this question on various channels.

## (Optional) What Type Of Content Change Is This? 
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## (Optional) Which Manual Does This Relate To?
- [ ] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [x] Desktop Client
- [ ] Other

**Comment by michaelstingl** [comment]
Quota is now displayed for every sync connection (4.0+)

**Comment by michaelstingl** [comment]
As user with Quota=1GB, who received a 100MB data from another user, I get this API response:

```
<?xml version="1.0"?>
<d:multistatus xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns" xmlns:s="http://sabredav.org/ns">
	<d:response>
		<d:href>/remote.php/dav/files/1GB-share-recipient/</d:href>
		<d:propstat>
			<d:prop>
				<d:quota-available-bytes>1073741824</d:quota-available-bytes>
				<d:quota-used-bytes>104857600</d:quota-used-bytes>
			</d:prop>
			<d:status>HTTP/1.1 200 OK</d:status>
		</d:propstat>
	</d:response>
</d:multistatus>
```

![CleanShot 2023-07-10 at 19 09 48](https://github.com/owncloud/docs-client-desktop/assets/214010/44822f5d-1805-419f-b135-9eae1bc9d293)

```
testpilotcloud 5.0.0.11221-daily20230611 078f59
Libraries Qt 6.4.3, OpenSSL 1.1.1t 7 Feb 2023
Using virtual files plugin: suffix
macos-22.5.0
```

**Comment by michaelstingl** [comment]
> Why do shares _recieved_ count against the quota?
> They do not take additional space for the recieving user except on his local drive.

Yes, it's about data usage on the local disc. Could be changed in future versions, because with VFS, local storage usage becomes less relevant.

> Note that the info I got from the ocis team tells that it does not count against...
> Or ist there a difference in oc10/ocis?

In **oC10**, received Shares and external storages are children of my WebDAV account root, so quota might be different in every folder.

In **oCIS**, Shares are a separate Space / sync connection.

**Comment by mmattel** [comment]
> In oC10, received Shares and external storages are children of my WebDAV account root, so quota might be different in every folder.

Means, the used quota differs per folder (or?)

> In oCIS, Shares are a separate Space / sync connection.

Ok, there is possibly a difference in oc10 vs ocis, pls also check the ocis API response. I would like to know what the recieved shares folder shows for quotas, if the response has similar effects than the one of oc10

> Yes, it's about data usage on the local disc.

This will be hard to get translated into a user understandable text. What happens if the used quota reaches max. Does it stop syncing? Because the user can still add real data to the server and the limit is just reached on the Desktop app. The actual limit is an artificial construct based on counting recieved shares (at least in oc10).
